backend/manifests/manifest-postgrest.yml

```diff
@@ -0,0 +1,15 @@
+---
+applications:
+  - name: ((pgrst_app_name))
+    instances: ((pgrst_instances))
+    disk_quota: ((pgrst_disk_quota))
+    memory: ((pgrst_mem_amount))
+    timeout: 180
+    docker:
+      image: ((pgrst_image))
+    routes:
+      - route: fac-((env_name))-postgrest.app.cloud.gov
```

Comment:
Should this also be ((app_name))?  (It shows up a few places... that is, if we set the app name, should we also use that app name in the URL?)

Reply:
@jadudm if we did that we should standardize across the fac app as well, but keep in mind, per Bret, the next step after this will be entirely removing postgrest and swagger from these manifests, and handling them in terraform

Reply:
I believe your proposed change is this, correct?:

`manifest-postgrest.yml`
```
    routes:
      - route: fac-((env_name))-postgrest.app.cloud.gov
```
 
 `manifest-swagger.yml`
```
    routes:
      - route: fac-((env_name))-((swagger_app_name)).app.cloud.gov
    env:
      API_URL: https://fac-((env_name))-((pgrst_app_name)).app.cloud.gov
      SWAGGER_JSON_URL: https://fac-((env_name))-((pgrst_app_name)).app.cloud.gov/
```

I don't see an issue with it if we want to go that abstract

Reply:
Ah. If this is going to go away soon with the move to Terraform, then don't worry about it either way. I was just pattern matching as I read through the PR.

Reply:
Personally I think we should only parameterize things that will change per-environment. Otherwise we're abstracting a "constant" and it just makes the manifest harder to read.

(Counterpoint: Making the app name fully abstract enables multiple parallel deployments in a given space without any collisions, eg if multiple people are working on a branch.)